### DISCUSSION TITLE
Question: about outlist

### DISCUSSION THREAD

**Discussion by niay000** [OpenFAST/openfast#1669] [Q&A]
Hi all;
I'm just getting started with OpenFAST. Please teach me. I’m using OpenFAST v3.0.0. My problem is the outlist section in the AeroDyn15.dat file can not use “All”. I want to input “All” to OutList. But then an error occurred. 
How should I resolve that?
Thank you.

Ayumi Nishimura

------- AERODYN v15.03.* INPUT FILE ------------------------------------------------
IEA 15 MW Offshore Reference Turbine
======  General Options  ============================================================================
False                  Echo        - Echo the input to "<rootname>.AD.ech"?  (flag)
"default"              DTAero      - Time interval for aerodynamic calculations {or "default"} (s)
          1            WakeMod     - Type of wake/induction model (switch) {0=none, 1=BEMT}
          2            AFAeroMod   - Type of blade airfoil aerodynamics model (switch) {1=steady model, 2=Beddoes-Leishman unsteady model} [must be 1 when linearizing]
          0            TwrPotent   - Type tower influence on wind based on potential flow around the tower (switch) {0=none, 1=baseline potential flow, 2=potential flow with Bak correction}
          0            TwrShadow   - Calculate tower influence on wind based on downstream tower shadow? (flag)
False                  TwrAero     - Calculate tower aerodynamic loads? (flag)
False                  FrozenWake  - Assume frozen wake during linearization? (flag) [used only when WakeMod=1 and when linearizing]
False                  CavitCheck  - Perform cavitation check? (flag) TRUE will turn off unsteady aerodynamics
False                  CompAA      - Flag to compute AeroAcoustics calculation [only used when WakeMod=1 or 2]
"unused"               AA_InputFile - Aeroacoustics input file
======  Environmental Conditions  ===================================================================
 1.225000000000000e+00 AirDens     - Air density (kg/m^3)
 1.479232653061225e-05 KinVisc     - Kinematic air viscosity (m^2/s)
 3.350000000000000e+02 SpdSound    - Speed of sound (m/s)
 1.035000000000000e+05 Patm        - Atmospheric pressure (Pa) [used only when CavitCheck=True]
 1.700000000000000e+03 Pvap        - Vapour pressure of fluid (Pa) [used only when CavitCheck=True]
 5.000000000000000e-01 FluidDepth  - Water depth above mid-hub height (m) [used only when CavitCheck=True]
======  Blade-Element/Momentum Theory Options  ====================================================== [used only when WakeMod=1]
          2            SkewMod     - Type of skewed-wake correction model (switch) {1=uncoupled, 2=Pitt/Peters, 3=coupled} [used only when WakeMod=1]
"default"              SkewModFactor - Constant used in Pitt/Peters skewed wake model {or "default" is 15/32*pi} (-) [used only when SkewMod=2; unused when WakeMod=0]
True                   TipLoss     - Use the Prandtl tip-loss model? (flag) [used only when WakeMod=1]
True                   HubLoss     - Use the Prandtl hub-loss model? (flag) [used only when WakeMod=1]
True                   TanInd      - Include tangential induction in BEMT calculations? (flag) [used only when WakeMod=1]
False                  AIDrag      - Include the drag term in the axial-induction calculation? (flag) [used only when WakeMod=1]
False                  TIDrag      - Include the drag term in the tangential-induction calculation? (flag) [used only when WakeMod=1 and TanInd=TRUE]
"Default"              IndToler    - Convergence tolerance for BEMT nonlinear solve residual equation {or "default"} (-) [used only when WakeMod=1]
        100            MaxIter     - Maximum number of iteration steps (-) [used only when WakeMod=1]
======  Dynamic Blade-Element/Momentum Theory Options  ====================================================== [used only when WakeMod=1]
          2            DBEMT_Mod   - Type of dynamic BEMT (DBEMT) model {1=constant tau1, 2=time-dependent tau1} (-) [used only when WakeMod=2]
          4            tau1_const  - Time constant for DBEMT (s) [used only when WakeMod=2 and DBEMT_Mod=1]
======  OLAF -- cOnvecting LAgrangian Filaments (Free Vortex Wake) Theory Options  ================== [used only when WakeMod=3]
"unused"               OLAFInputFileName - Input file for OLAF [used only when WakeMod=3]
======  Beddoes-Leishman Unsteady Airfoil Aerodynamics Options  ===================================== [used only when AFAeroMod=2]
          3            UAMod       Unsteady Aero Model Switch (switch) {1=Baseline model (Original), 2=Gonzalez's variant (changes in Cn,Cc,Cm), 3=Minemma/Pierce variant (changes in Cc and Cm)} [used only when AFAeroMod=2]
True                   FLookup     Flag to indicate whether a lookup for f' will be calculated (TRUE) or whether best-fit exponential equations will be used (FALSE); if FALSE S1-S4 must be provided in airfoil input files (flag) [used only when AFAeroMod=2]
======  Airfoil Information =========================================================================
          1            AFTabMod    - Interpolation method for multiple airfoil tables {1=1D interpolation on AoA (first table only); 2=2D interpolation on AoA and Re; 3=2D interpolation on AoA and UserProp} (-)
          1            InCol_Alfa  - The column in the airfoil tables that contains the angle of attack (-)
          2            InCol_Cl    - The column in the airfoil tables that contains the lift coefficient (-)
          3            InCol_Cd    - The column in the airfoil tables that contains the drag coefficient (-)
          4            InCol_Cm    - The column in the airfoil tables that contains the pitching-moment coefficient; use zero if there is no Cm column (-)
          0            InCol_Cpmin - The column in the airfoil tables that contains the Cpmin coefficient; use zero if there is no Cpmin column (-)
         50            NumAFfiles  - Number of airfoil files used (-)
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_00.dat"    AFNames            - Airfoil file names (NumAFfiles lines) (quoted strings)
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_01.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_02.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_03.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_04.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_05.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_06.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_07.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_08.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_09.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_10.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_11.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_12.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_13.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_14.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_15.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_16.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_17.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_18.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_19.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_20.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_21.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_22.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_23.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_24.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_25.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_26.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_27.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_28.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_29.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_30.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_31.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_32.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_33.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_34.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_35.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_36.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_37.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_38.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_39.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_40.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_41.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_42.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_43.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_44.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_45.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_46.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_47.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_48.dat"
"Airfoils/IEA-15-240-RWT_AeroDyn15_Polar_49.dat"
======  Rotor/Blade Properties  =====================================================================
True                   UseBlCm     - Include aerodynamic pitching moment in calculations?  (flag)
"aerodyn15_blade.dat" ADBlFile(1) - Name of file containing distributed aerodynamic properties for Blade #1 (-)
"aerodyn15_blade.dat" ADBlFile(2) - Name of file containing distributed aerodynamic properties for Blade #2 (-) [unused if NumBl < 2]
"aerodyn15_blade.dat" ADBlFile(3) - Name of file containing distributed aerodynamic properties for Blade #3 (-) [unused if NumBl < 3]
======  Tower Influence and Aerodynamics ============================================================= [used only when TwrPotent/=0, TwrShadow=True, or TwrAero=True]
11                     NumTwrNds   - Number of tower nodes used in the analysis  (-) [used only when TwrPotent/=0, TwrShadow=True, or TwrAero=True]
TwrElev        TwrDiam        TwrCd          TwrTI (used only with TwrShadow=2)
(m)              (m)           (-)            (-)
  15.           10.             1.          1.0000000E-01
  28.           10.             1.          1.0000000E-01
  41.            9.926          1.          1.0000000E-01
  54.            9.443          1.          1.0000000E-01
  67.            8.833          1.          1.0000000E-01
  80.            8.151          1.          1.0000000E-01
  93.            7.39           1.          1.0000000E-01
 106.            6.909          1.          1.0000000E-01
 119.            6.748          1.          1.0000000E-01
 132.            6.572          1.          1.0000000E-01
 144.495         6.5            1.          1.0000000E-01
======  Tower Influence and Aerodynamics ============================================================= [used only when TwrPotent/=0, TwrShadow=True, or TwrAero=True]
True                   SumPrint    - Generate a summary file listing input options and interpolated properties to "<rootname>.AD.sum"?  (flag)
            0          NBlOuts     - Number of blade node outputs [0 - 9] (-)
1, 6, 11, 20, 30, 38, 43, 47, 50 BlOutNd     - Blade nodes whose values will be output  (-)
            0          NTwOuts     - Number of tower node outputs [0 - 9]  (-)
1, 2, 3, 4, 5          TwOutNd     - Tower nodes whose values will be output  (-)
                       OutList     - The next line(s) contains a list of output parameters.  See OutListParameters.xlsx for a listing of available output channels, (-)
"All"                            - rotor-tip speed ratio (-)
END of input file (the word "END" must appear in the first 3 columns of this last OutList line)
---------------------------------------------------------------------------------------

**Comment by jjonkman** [comment]
Dear @niay000,

"All" is not an available output parameter from AeroDyn v15 and there is no option in AeroDyn to output everything (and you probably wouldn't want to consider the large amount of data that would be generated).  The list of output parameters that you can specify in the AeroDyn `OutList` is documented here: https://openfast.readthedocs.io/en/main/source/user/aerodyn/appendix.html#aerodyn-list-of-output-channels and https://openfast.readthedocs.io/en/main/_downloads/3f19498a5dc774461e022b671ff01ec6/OutListParameters.xlsx.

You can also output all blade-related quantities through the nodal outputs available through AeroDyn, as documented here: https://openfast.readthedocs.io/en/main/source/user/aerodyn/input.html#nodal-outputs.

Best regards,

**Reply by niay000** [reply]
Dear　jjonkman
Thank you for your reply. I understand very well. I'll try it right away.
Thank you,
